exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -371,8 +386,42 @@ func (p *Poloniex) GetFundingHistory() ([]exchange.FundHistory, error) {
 }
 
 // GetExchangeHistory returns historic trade data since exchange opening.
-func (p *Poloniex) GetExchangeHistory(currencyPair currency.Pair, assetType asset.Item) ([]exchange.TradeHistory, error) {
-	return nil, common.ErrNotYetImplemented
+func (p *Poloniex) GetExchangeHistory(req *trade.HistoryRequest) ([]trade.History, error) {
+	if req.TimestampStart.Unix() == 0 {
```

Comment:
In another wrapper function, you left a comment that you can't retrieve data older than X. For poloniex, you cannot get data older than a year, just a note